app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Expenses</h1>
+
+<div id="expenses">
+  <% @expenses.each do |expense| %>
+    <%= render expense %>
+    <p>
+      <%= link_to "Show this expense", expense %>
+    </p>
+  <% end %>
+</div>
+
```

Comment:
- [ ] Please make sure the list of transactions(expenses) is presented, ordered by the most recent as requested in the project requirements. but it's presented, ordered by the least recent. Please make sure to fix this issue.

![image](https://github.com/Othniel2471/budget_app/assets/83803414/85b63807-6ebd-4a88-8f34-22870b875f3d)